tests/backend/http/test_http.py

```diff
@@ -104,6 +104,19 @@ async def assert_stream_closed_on_peer_side(stream):
         await stream.send_all(b"GET / HTTP/1.0\r\n\r\n")
 
 
+@customize_fixtures(use_enforce_https=True)
+@pytest.mark.trio
+async def test_enforce_https(backend, backend_http_send):
+    assert backend.config.enforce_https is True
```

Comment:
no need to check this given it is the responsibility of `customize_fixture` to enable this stuff (and if it is not enabled, the test should fail by itself)